src/App.tsx

```diff
@@ -1,61 +1,115 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
The `usersFromServer` should not be directly imported here. Instead, consider fetching or preparing this data in the `App` component and passing it down as props to maintain a single source of truth.